lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
If they use the explicit `timestamp(...)` validator, the error message returned is explicit to that type stating _"number of milliseconds"_ or _"number of seconds"_.